releasenotes/notes/logs-docker-kubernetes-format-fix-8f0101796f2fe46c.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
```

Comment:
should we remove this comment?

Reply:
it's removed automatically when re-aggregating release notes, no need